### PR TITLE
Fix example checker validation for incomplete multifile examples

### LINKED ISSUE #24994: [Tour of Beam] [Bug] Example checker fails when verifying incomplete multifile examples
### What needs to happen?
Example CI check is broken for incomplete multifile examples
Here is a broken workflow example:
https://github.com/apache/beam/actions/runs/3910959939/jobs/6683858591

### Priority: 
2 (default / most normal work should be filed as P2)

### Issue Components

- [ ]  Component: Python SDK
- [ ]  Component: Java SDK
- [ ]  Component: Go SDK
- [ ]  Component: Typescript SDK
- [ ]  Component: IO connector
- [ ]  Component: Beam examples
- [x]  Component: Beam playground
- [ ]  Component: Beam katas
- [ ]  Component: Website
- [ ]  Component: Spark Runner
- [ ]  Component: Flink Runner
- [ ]  Component: Samza Runner
- [ ]  Component: Twister2 Runner
- [ ]  Component: Hazelcast Jet Runner
- [ ]  Component: Google Cloud Dataflow Runner

### PR DESCRIPTION
**Pull request by olehborysevych**
Fixing checker for incomplete multifile examples
fixes #24994
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: playground/infrastructure/helper.py
Comment (TSultanov) [inline playground/infrastructure/helper.py:194]: What's the type of `filepath` that we need to cast it into a string?

**Comment by olehborysevych** [comment]
R: @damccorm

File: playground/infrastructure/helper.py
Comment (damccorm) [inline playground/infrastructure/helper.py:204]: If this condition isn't true `err.errors()[0]["msg"] == "multifile is True but no files defined"` we still want to raise an exception, right?

Should this just be:

```
        if len(err.errors()) == 1 and err.errors()[0]["msg"] == "multifile is True but no files defined":
            logging.warning("incomplete multifile example ignored %s", filepath)
            return None
        raise
```

Comment (olehborysevych) [reply playground/infrastructure/helper.py:202]: Good catch! Thank you!

**Review by damccorm — APPROVED** [review]
LGTM - I'll wait for CI to pass before merging though

**Pull request merged**
